css-four-column-layout/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Four Column Layout</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+
+      <div class="row">
+        <h2>How It Works</h2>
+      </div>
+      <div class="row">
+        <div column = "column-25-percent">
+          <img src="images/screen-1.png" alt="app switching" class="app-pic">
+          <p></p>
```

Comment:
Make sure to remove these empty elements. if you need the margin, some bottom margin on the above element can take care of that for you!